sycl/include/sycl/handler.hpp

```diff
@@ -1457,8 +1457,8 @@ class __SYCL_EXPORT handler {
   template <typename SharedPtrT>
   void setHandlerKernelBundle(SharedPtrT &&NewKernelBundleImpPtr);
 
-  void SetHostTask(std::function<void()> &&Func);
-  void SetHostTask(std::function<void(interop_handle)> &&Func);
+  void SetHostTask(std::function<void()> Func);
+  void SetHostTask(std::function<void(interop_handle)> Func);
```

Comment:
`std::function` is used for type-erasure, so "by value" semantics looks correct to me.